spec/models/rent_spec.rb

```diff
@@ -2,35 +2,35 @@
 require 'rails_helper'
 
 RSpec.describe Rent, type: :model do
-  subject(:rent) { FactoryGirl.build(:rent) }
+  let(:rent) { FactoryGirl.build(:rent) }
 
   describe :Associations do
-    it { should belong_to(:profile) }
-    it { should belong_to(:council) }
+    it { expect(rent).to belong_to(:profile) }
```

Comment:
I might agree with you there ;) 
https://github.com/fsek/web/blob/auth-test/spec/controllers/election/candidates_controller_spec.rb#L79
How would a block expression look with ```should```?